spaces/S000146/properties/P000133.md

```diff
@@ -0,0 +1,12 @@
+---
+space: S000146
+property: P000133
+value: true
+refs:
+  - zb: "0642.54033"
+    name: Characterizations of the countable infinite product of rationals and some related problems
+---
+
+In fact, {S146} can be viewed as a subspace of $\omega^\omega$ which $\lim_{n \to + \infty} x_n = \infty$, by {{zb:0642.54033}} (<https://eudml.org/doc/220890>).
+
+Then using the embedding of $\omega^\omega$ into $\mathbb R$ ({S28}), we know that {S146} can be viewed as a dense subspace of $\mathbb R$, which is {P133}.
```

Comment:
```suggestion
The result then follows as $\omega^\omega$ is homeomorphic to {S28}, and {S28|P133}.
```

Reply:
Oops, hang on; this is only enough for GO-space...

Reply:
Well because {P133} (LOTS) is not directly hereditarily, my argument is that:
If $A$ is embeddable into $\mathbb R$ and the embedding is dense in $\mathbb R$, then $A$ is LOTS.
Otherwise it is not always true, e.g., $A = (0, 1) \cup \{2\}$, it is the counterexample of {S198}.

So it is not so simple and that's the reason I hanged.

Reply:
I think I agree with you. I couldn't find any proof that a dense subspace of a LOTS is LOTS on MSE, so that's a good question to ask and answer there. So I think we should remove this assertion for now, and it can be added after a satisfactory MSE post.